### PR TITLE
Fixed readme + made config script more user friendly

### PR DESCRIPTION
**Pull request by Danek309044**
## Changes:
- The configuration script will ask you if you want to run a selfbot right away.
- `.env` file will be created with the needed environments.
- Fixed readme (like the step 3. dissaperence)

### CODE REVIEW

**Comment by Danek309044** [comment]
Also i tested the config script and all worked how it should had.

File: clyde.sh [hunk 1]
```diff
@@ -2,14 +2,37 @@
 shopt -s extglob
 
 if [ ! -d .venv ]; then
+  echo ""
   echo "Configuring Clyde..."
```
Comment (celunah) [inline clyde.sh:6]: Running just `echo` will also print a newline.

File: clyde.sh [hunk 2]
```diff
@@ -2,14 +2,37 @@
...
+    Y | y)
+        echo "Making a selfbot..."
+        ln -sf clyde.selfbot.py clyde.py
+        ;;
+    *)
+        echo "Continuing to make a bot..."
+        ln -sf clyde.bot.py clyde.py
+        ;;
+  esac
+fi
```
Comment (piotr25691) [inline clyde.sh:25]: This is a cleaner `if` syntax you can use.

File: clyde.sh [hunk 3]
```diff
@@ -2,14 +2,37 @@
...
+
+if [ ! -f .env ]; then
+  echo "Creating .env file..."
+  echo "TOKEN=\"<TOKEN_GOES_HERE>\"" >> .env
+  echo "OWNER=<YOUR_DISCORD_ID_GOES_HERE>" >> .env
+  echo "ERROR_CHANNEL=<ID_OF_ERROR_LOGGING_CHANNEL_GOES_HERE>" >> .env
+  echo ""
+  echo -e "\033[0;31mDon't forget to configure the .env file!\033[0m"
```
Comment (piotr25691) [inline clyde.sh:33]: Single-quoted strings also exist.

**Review by celunah — CHANGES_REQUESTED** [review]

File: clyde.sh (hunk 2 above)
Comment (piotr25691) [inline clyde.sh:25]:
```sh
if [[ $REPLY =~ ^[Yy]$ ]]; then
    echo "Making a selfbot..."
    ln -sf clyde.selfbot.py clyde.py
else
    echo "Continuing to make a bot..."
    ln -sf clyde.bot.py clyde.py
fi
```
would work better here.

File: clyde.sh [hunk 4]
```diff
@@ -2,14 +2,37 @@
...
+        ln -sf clyde.bot.py clyde.py
+        ;;
+  esac
+fi
+
+if [ ! -f .env ]; then
+  echo "Creating .env file..."
+  echo "TOKEN=\"<TOKEN_GOES_HERE>\"" >> .env
+  echo "OWNER=<YOUR_DISCORD_ID_GOES_HERE>" >> .env
+  echo "ERROR_CHANNEL=<ID_OF_ERROR_LOGGING_CHANNEL_GOES_HERE>" >> .env
```
Comment (celunah) [inline clyde.sh]: Have you heard of single-quoted strings yet?

**Review by celunah — CHANGES_REQUESTED** [review]

File: clyde.sh (hunk 4 above)
Comment (Danek309044) [reply clyde.sh]: yeah...

**Review by celunah — APPROVED** [review]
good enough

File: clyde.sh (hunk 1 above)
Comment (Danek309044) [reply clyde.sh:6]: i know

**Review by Danek309044 — COMMENTED** [review]
Edited it.

**Pull request merged**